### PR TITLE
Temporal Nexus: Self Hosted and Enable Nexus

### PR DESCRIPTION
**Pull request by jsundai**
## What does this PR do?
[Preview ](https://temporal-documentation-git-self-host-nexus-pg.preview.thundergun.io/production-deployment/self-hosted-guide/enable-nexus)

### CODE REVIEW

File: docs/production-deployment/self-hosted-guide/enable-nexus.mdx [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+  - self-hosted
+  - enable-nexus
+---
+
+Nexus is only supported in single cluster setups for the time being due to endpoint registry replication not being implemented yet. 
+Enable Nexus in your self-hosted Temporal Service by updating the server's static configuration file and enabling Nexus through dynamic config, then setting the public callback URL and allowed callback addresses.
+
+## Enabling Nexus
+
+To enable Nexus in your deployment:
```
Comment (bergundy) [inline docs/production-deployment/self-hosted-guide/enable-nexus.mdx:19]:
```suggestion
To enable Nexus in your deployment:

> NOTE: Replace `$PUBLIC_URL` with a URL value that is accessible to external callers or internally within the cluster. At the time of writing external Nexus calls are considered experimental so it should be safe to use the address of an internal load balancer for the frontend service.
```

Comment (bergundy) [reply docs/production-deployment/self-hosted-guide/enable-nexus.mdx:19]: I should call this out in the markdown file in the server repo.

**Review by prasek — COMMENTED** [review]
lgtm with @bergundy's suggestions

File: docs/production-deployment/self-hosted-guide/enable-nexus.mdx (hunk 1 above)
Comment (bergundy) [reply docs/production-deployment/self-hosted-guide/enable-nexus.mdx:19]: https://github.com/temporalio/temporal/pull/6660

**Comment by axfelix** [comment]
Also LGTM!

**Review by axfelix — APPROVED** [review]

**Pull request merged**